### PR TITLE
[Compute] WriteAccelerator feature

### PR DESCRIPTION
**Pull request by hyonholee**
Add WriteAcceleratorEnabled parameter to Set-AzureRmVMOSDisk, Set-AzureRmVMDataDisk, Add-AzureRmVMDataDisk, Set-AzureRmVmssStorageProfile, Add-AzureRmVmssDataDisk and Update-AzureRmVmss cmdlets.

https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/12

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#making-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by maddieclayton** [comment]
@hyonholee Added a comment to your cmdlet review, please let me know once this comment is addressed and I'll take a look at your PR.

File: src/ResourceManager/Compute/Commands.Compute/VirtualMachine/Config/AddAzureVMDataDiskCommand.cs [hunk 1]
```diff
@@ -105,6 +105,11 @@ public class AddAzureVMDataDiskCommand : Microsoft.Azure.Commands.ResourceManage
         [ValidateNotNullOrEmpty]
         public StorageAccountTypes? StorageAccountType { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = false)]
+        public bool? WriteAcceleratorEnabled { get; set; }
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/VirtualMachine/Config/AddAzureVMDataDiskCommand.cs]: I thought for this cmdlet we were going to use a switch

File: src/ResourceManager/Compute/Commands.Compute/VirtualMachine/Config/SetAzureVMDataDiskCommand.cs [hunk 2]
```diff
@@ -82,6 +82,11 @@ public class SetAzureVMDataDiskCommand : Microsoft.Azure.Commands.ResourceManage
         [ValidateNotNullOrEmpty]
         public StorageAccountTypes? StorageAccountType { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = false)]
+        public bool? WriteAcceleratorEnabled { get; set; }
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/VirtualMachine/Config/SetAzureVMDataDiskCommand.cs]: switch

File: src/ResourceManager/Compute/Commands.Compute/VirtualMachine/Config/SetAzureVMOSDiskCommand.cs [hunk 3]
```diff
@@ -163,6 +163,11 @@ public class SetAzureVMOSDiskCommand : Microsoft.Azure.Commands.ResourceManager.
         [ValidateNotNullOrEmpty]
         public StorageAccountTypes? StorageAccountType { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = false)]
+        public bool? WriteAcceleratorEnabled { get; set; }
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/VirtualMachine/Config/SetAzureVMOSDiskCommand.cs]: switch

**Comment by markcowl** [comment]
on demand run here: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/powershell-demand/531/

**Comment by cormacpayne** [comment]
On-demand: https://azuresdkci.westus2.cloudapp.azure.com/job/powershell-demand/534/

**Comment by cormacpayne** [comment]
@hyonholee the build is failing because the version of the Compute SDK consumed in the ServiceFabric project needs to be updated:
- [`Commands.ServiceFabric.csproj`](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/ServiceFabric/Commands.ServiceFabric/Commands.ServiceFabric.csproj#L155-L157)
- [`Commands.ServiceFabric/packages.config`](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/ServiceFabric/Commands.ServiceFabric/packages.config#L5)
- [`Commands.ServiceFabric.Test.csproj`](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/ServiceFabric/Commands.ServiceFabric.Test/Commands.ServiceFabric.Test.csproj#L82-L84)
- [`Commands.ServiceFabric.Test/packages.config`](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/ServiceFabric/Commands.ServiceFabric.Test/packages.config#L9)

In an upcoming sprint, we're looking at create a new project for Compute common code that multiple projects can depend on rather than the Compute SDK, which will mitigate this issue.

**Comment by cormacpayne** [comment]
On-demand: https://azuresdkci.westus2.cloudapp.azure.com/job/powershell-demand/539/

**Pull request merged**